scripts/userstyles.yml

```
@@ -429,6 +431,13 @@ userstyles:
     readme:
       app-link: https://news.ycombinator.com
     current-maintainers: [*lucasmelin]
+  hardcover:
+    name: Hardcover
+    categories: [library, social_networking]
```

```suggestion
    categories: [productivity, social_networking]
```
The library category is unfortunately not for things relating to actual book libraries, it is for software libraries 😅